database/model.py

```diff
@@ -0,0 +1,148 @@
+from sqlalchemy import Boolean, Column, ForeignKey, Integer, String, Date, DateTime

```

Comment:
chyba dużo importów jest tutaj nieużywane

Reply:
Konkretnie dwa - poprawione.

Reply:
![image](https://user-images.githubusercontent.com/26205848/145230893-3182f62e-68c4-4510-ab4c-f68500e90c37.png)

Reply:
z tego co widzę to jeszcze `sessionMaker`, `UniqueConstraint` i `declarative_base`

Reply:
Fakt. Myślałem, że chodzi tylko o ten jeden wiersz.